examples/matching_engine/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub struct Price {
     pub price: u64,
 }
 
+/// We use the custom serialization for the Price so that the order of the serialization
+/// corresponds to the order of the 
```

Comment:
Did I forget to complete the sentence in my proposal. :sweat_smile:

Reply:
The error was mine of course.